### ISSUE TITLE
[1] a11y: add affordance for screen readers when form has been submitted

### ISSUE THREAD

**Issue by karrui** [opengovsg/FormSG#4189]
Currently, the screen changes to the submission success page without any announcements for screen readers.
Screen reader users may feel confused with the lack of success feedback.

Suggest focusing on success text when loaded so screen readers will read it out. Or add aria declaration "Your response has been submitted"

**Comment by karrui** [comment]
This was actually closed by #4917.
